packages/components/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
```

Comment:
Is there any reason to keep two different tsconfigs? All projects should support `import` syntax even if they don't tree shake it.